workshop/content/20-typescript/70-advanced-topics/200-pipelines/3000-new-pipeline.md

```diff
@@ -6,7 +6,7 @@ weight = 130
 ## Define an Empty Pipeline
 Now we are ready to define the basics of the pipeline.
 
-We will be using a new package here, so first `npm install aws-cdk-lib/pipelines`.
+We will be using a new package here, so first `npm install @aws-cdk-lib/pipelines`.
```

Comment:
I addressed this in another PR. The correct thing to do is to not install any new package because pipelines is in the aws-cdk-lib package already

Reply:
My bad, first time doing this. Just updated